snippets/hook_consumer.code-snippets

```diff
@@ -1,17 +1,37 @@
 {
-  "Hook Consumer Widget": {
+  "Hook Consumer Stateless Widget": {
     "scope": "dart",
-    "prefix": "hookConsumer",
+    "prefix": "hookStlessConsumer",
```

Comment:
We might consider renaming it to `stlessHookConsumer` to better keep in line with the other widget prefixes